exercises/practice/complex-numbers/lib/complex_numbers.ex

```diff
@@ -1,6 +1,6 @@
 defmodule ComplexNumbers do
   @typedoc """
-  In this module, complex numbers are represented as a tuple-pair containing the real and
+  In this module, complex numbers, or a real and a complex number are represented as a tuple-pair containing the real and
```

Comment:
This addition doesn't sound right. Real numbers aren't represented as tuples

Reply:
Oh, this is a search and replace accident. Sorry about that.
```suggestion
  In this module, complex numbers are represented as a tuple-pair containing the real and
```